add 'make package' libadb target to make libadb packages

mykmelez opened the pull request:
@ochameau, can you take a look at this? It should be straightforward: a `make package` target for libadb (so we don't have to construct these packages by hand).

mykmelez committed:
add 'make package' libadb target to make libadb packages

ochameau commented on android-tools/adb-bin/Makefile:
```diff
@@ -3,15 +3,24 @@
+  PACKAGE_FILES += \
+    ../win-out/libadb.dll \
+    ../adb-win-api/api/objfre_wxp_x86/i386/AdbWinApi.dll \
+    ../adb-win-api/winusb/objfre_wxp_x86/i386/AdbWinUsbApi.dll
```
Wouldn't it be better to use `obj*` instead of `objfre_wxp_x86`, like what is being done for `WINDOWS_ARTIFACTS`?

ochameau commented:
Ship it!

mykmelez committed:
ignore paths when packaging libadb files

mykmelez committed:
build only libadb by default on all platforms

mykmelez committed:
use wildcard to identify obj* dirs for DLLs

mykmelez commented:
> Wouldn't it be better to use obj\* instead of objfre_wxp_x86, like what is being done for WINDOWS_ARTIFACTS?

Yes, it would! Fixed in ec74058c6050cf98d74009d262dde74fd66910b1.

I also fixed a couple other issues:
- Use the `-j` flag to _zip_ to trim directory paths from the files being packaged (Windows-specific, as only Windows has multiple source dirs and builds into separate object dirs).
- Build the same thing (libadb) by default on all three platforms.

mykmelez commented:
> Ship it!

À votre demande!
